packages/cli/src/util/build/write-build-result.ts

```diff
@@ -385,7 +385,7 @@ export async function* findDirs(
   }
   for (const path of paths) {
     const abs = join(dir, path);
-    const s = await fs.stat(abs);
+    const s = await fs.lstat(abs);
```

Comment:
Do we still want the try/catch for the ENOENT case?

Reply:
It shouldn't really happen now, but ya sure, I'll add it just because this piece of logic should never make the build fail.

Reply:
da0e8eade299cdde3126020e279fc53e064cccf3